src/CMakeLists.txt

```diff
@@ -200,8 +200,8 @@ endif()
 # Python stuff
 find_package(PythonInterp 2 REQUIRED)
 find_package(PythonLibs 2 REQUIRED)
-find_package(Python3Interp 3 REQUIRED)
-find_package(Python3Libs 3 REQUIRED)
+find_package(Python3Interp 3 QUIET)
+find_package(Python3Libs 3 QUIET)
```

Comment:
@tchaikov 
I think it is the other way around.
With QUIET the absence of python3 stuff is sillently ignored.

If I test with REQUIRED, things seriously blow up in my face on some Jenkins runs where there is python3 to be installed. And things really stop during CMake processing.
So I added this to actually get some testing. (Now it stops on the last test due to timeout, and unrelated to this commit.)
But perhaps somebody fixed the python stuff in the mean time.
I'll drop this extra commit.